heartbeat/monitors/active/http/task.go

```diff
@@ -41,25 +42,25 @@ func newHTTPMonitorHostJob(
 		return nil, err
 	}
 
-	hostname, port, err := splitHostnamePort(request)
+	hostname, _, err := splitHostnamePort(request)
```

Comment:
Port is missing here. This constructor is used when a proxy is configured + name lookup happens by proxy itself. I did undo the tcp dialer to report the port number due to potential use of proxies.